src/Microsoft.Health.Fhir.Api/Registration/FhirServerApplicationBuilderExtensions.cs

```diff
@@ -39,8 +51,46 @@ public static IApplicationBuilder UseFhirServer(this IApplicationBuilder app, Fu
                 app.UseMiddleware<PathBaseMiddleware>(pathString);
             }
 
+            app.UseHttpsRedirection();
             app.UseStaticFiles();
-            app.UseMvc();
+
+            app.UseRouting();
```

Comment:
I believe these changes should also be added to Health Paas, correct?

Something I was thinking about: in this part of the code we are using the Default CORS policy, but customers in Health Paas and OSS can override the CORS configuration. Are we not overriding any possible custom configuration?

Reply:
Good catch! Undo the cors registration here and added it back to OSS StartupFilter as previously implemented.